src/app/app.module.ts

```diff
@@ -1,17 +1,20 @@
+/* Angular specific imports */
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms'; // <-- NgModel lives here
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```

Comment:
I'd like to see all the material modules in their own separate module and then imported to app.module.ts, but we can address that in another pull request :)